src/background.js

```diff
@@ -18,6 +19,49 @@ import store from "./media-manager/store";
 
 const isDevelopment = process.env.NODE_ENV !== "production";
 
+// We need to ask macOS permission to access media devices
+async function getMediaPermission() {
```

Comment:
I would move this function underneath the `checkMediaPermission` as it would be nice when you read the code and the call order is also represented in the code order.